phpcs.xml.dist

```diff
@@ -2,6 +2,9 @@
 <ruleset name="es-wp-query">
 	<description>Sniffs for the coding standards of the ES WP Query plugin</description>
 
+    <!-- Include Alley Rules -->
+    <rule ref="Alley-Interactive" />
```

Comment:
Can you add `<config name="testVersion" value="7.4-" />` too?

Reply:
added!